### PR TITLE
Intern Task Alina Mulik: Homework

### PR DESCRIPTION
**Pull request by alina-mulik**
adding homework 7

### CODE REVIEW

File: Classes7.1/LivingBeingAbstractClass.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿namespace Classes7._1
+{
+    public abstract class LivingBeingAbstractClass
+    {
+        public bool isHappy;
+        public int pointsOfHappiness;
+        public static int counterOfLivingBeings;
```
Comment (yauhenists) [inline Classes7.1/LivingBeingAbstractClass.cs]: Not a mistake of the task, but:
1. static field name - starts with upper case letter
2. fields - it's better to make them protected. Name for protected - starts with upper case letter

File: Classes7.2/IHero.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Classes7._2
+{
+    interface IHero
+    {
+        public const string HeroMotto = "I have no fear!";
+        public int NumberOfLives { get; set; }
+        public bool isGood { get; }
```
Comment (yauhenists) [inline Classes7.2/IHero.cs]: Property - starts with upper case letter

File: Classes7.2/AntiHero.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        private int _numberOfLives;
+        private string _weapon;
+        public int NumberOfLives { get => _numberOfLives; set => _numberOfLives = value; }
+        public bool isGood { get => false; }
+        public string Weapon { get => _weapon; set => _weapon = value; }
+
+        public AntiHero(int numberOfLives, string weapon)
+        {
+            NumberOfLives = numberOfLives;
+            weapon = _weapon;
```
Comment (yauhenists) [inline Classes7.2/AntiHero.cs]: I guess this is mistake) _weapon = weapon

File: Classes7.3/IPrint.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Classes7._3
+{
+    public interface IPrint
+    {
+        public string Description { get;}
```
Comment (yauhenists) [inline Classes7.3/IPrint.cs]: You can remove from here and make this property virtual in abstract class

File: Classes7.3/DeviceAbstractClass.cs [hunk 5]
```diff
@@ -0,0 +1,47 @@
+﻿namespace Classes7._3
+{
+    public abstract class DeviceAbstractClass
+    {
+        public double numberOfPixelsInCamera;
```
Comment (yauhenists) [inline Classes7.3/DeviceAbstractClass.cs]: This field is not used in Printer class. You can move this as property to ITakePhoto interface

Comment (yauhenists) [reply Classes7.3/DeviceAbstractClass.cs]: The same with paperWidth, paperHeight

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: Classes7.1/LivingBeingAbstractClass.cs (hunk 1 above)
Comment (alina-mulik) [reply Classes7.1/LivingBeingAbstractClass.cs]: Got it, thanks! Fixed.

File: Classes7.2/IHero.cs (hunk 2 above)
Comment (alina-mulik) [reply Classes7.2/IHero.cs]: Fixed.

File: Classes7.2/AntiHero.cs (hunk 3 above)
Comment (alina-mulik) [reply Classes7.2/AntiHero.cs]: Oops, fixed it. 🥲

File: Classes7.3/IPrint.cs (hunk 4 above)
Comment (alina-mulik) [reply Classes7.3/IPrint.cs]: Got it, reorganized this project a bit. Fixed.

File: Classes7.3/DeviceAbstractClass.cs (hunk 5 above)
Comment (alina-mulik) [reply Classes7.3/DeviceAbstractClass.cs]: Fixed both 😅

**Review by yauhenists — APPROVED** [review]

**Pull request merged**